src/Uno/Prism.Uno/Services/Dialogs/DialogWindow.xaml

```diff
@@ -0,0 +1,11 @@
+﻿<ContentDialog x:Class="Prism.Services.Dialogs.DialogWindow"
+               xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+               xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+               xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+               xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+               mc:Ignorable="d"
+               Title="Dialog"
+               Height="450"
+               Width="800">
+
+</ContentDialog>
```

Comment:
Please see the WPF Implementation for the DialogWindow... This shouldn't hardcode the Title/Height/Width... Also the DialogWindow would be the default type if not specified. https://github.com/PrismLibrary/Prism/blob/master/src/Wpf/Prism.Wpf/Services/Dialogs/DialogWindow.xaml

Reply:
> Please see the WPF Implementation for the DialogWindow... This shouldn't hardcode the Title/Height/Width... 

Agreed.

> Also the DialogWindow would be the default type if not specified. https://github.com/PrismLibrary/Prism/blob/master/src/Wpf/Prism.Wpf/Services/Dialogs/DialogWindow.xaml

I do not understand what that means. The default dialog is registered here, if I understand correctly: 

https://github.com/PrismLibrary/Prism/blob/4a109d697eeeeef3274bff1c936c186616c6bf71/src/Wpf/Prism.Wpf/PrismInitializationExtensions.cs#L40

What change would you make ?